version.xml

```diff
@@ -1,5 +1,5 @@
 <Project>
   <PropertyGroup>
-    <PatchVersion>5</PatchVersion>
+    <PatchVersion>6</PatchVersion>
```

Comment:
just keep 5

I'll do the version update in a separate commit

Reply:
Thx for the input, I fixed it.